tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
+envlist =
+    py27,
+    py35,
+    py36,
+    py37,
+    py38,
+    pypy
+    pypy3
+
+[testenv]
+commands =
+    python setup.py test
```

Comment:
`python setup.py anything` is deprecated - at least that is what I read like a year ago in a Twitter thread of one of the members of pypa.

Instead of using `py.test` (which I use exclusively) you could also use Python`s unitest module as following...
`python -m unittest discover -s plone/synchronize`

Reply:
Ah, yes, I had heard/read the same somewhere. Well, at least about `python setup.py test` being deprecated.
I thought about `py.test`, but your command looks useful, although a bit verbose. Thanks for the tip.